prow/git/git.go

```diff
@@ -423,6 +425,21 @@ func (r *Repo) mergeWithMergeStrategySquash(commitlike string) (bool, error) {
 	return true, nil
 }
 
+func (r *Repo) mergeWithMergeStrategyRebase(commitlike string) (bool, error) {
+	co := r.gitCommand("rebase", "--no-stat", commitlike)
```

Comment:
This rebases the basebranch on top of the feature branch, I think what github does is rebasing the feature branch on top of the base branch